src/ProgressiveColonizationSystem/PksOnStationParameterFactory.cs

```diff
@@ -0,0 +1,107 @@
+﻿using ContractConfigurator;
+using Contracts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace ProgressiveColonizationSystem
+{
+    /// <summary>
+    ///  Use this if it seems like a good idea to require that the kerbal make it all the way
+    ///  to a base.  Otherwise use the ReachState configuration.
+    /// </summary>
+    public class PksOnStationParameterFactory
+        : ParameterFactory
+    {
+        private string body;
+        private string researchCategory;
+        private int tier;
+        private string kerbal;
+
+        public override bool Load(ConfigNode configNode)
```

Comment:
Is this loading from the configuration?  Or from a quicksave?  I'm kinda hoping it's loading from the config, but I don't believe it is...

Reply:
This is ParameterType, so from the config.  The Parameter class is what is loaded from the save.